sgv2-docsapi/pom.xml

```diff
@@ -152,6 +152,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
+        <executions>
+          <execution>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <delete>
+                  <fileset dir="${project.build.directory}/generated-sources/grpc/io/stargate/bridge/proto">
+                    <include name="QueryOuterClass.java"/>
+                    <include name="Schema.java"/>
+                  </fileset>
+                </delete>
+              </target>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Can we just exclude the generated classes from the `common`, already this this for #1720 [work](https://github.com/stargate/stargate/tree/ise-1720-table-manager-schema) and all works fine, tests are green.. I am building queries, sending them over the bridge and it's all fine.. My setup is the following:

```
    <dependency>
      <groupId>io.stargate.grpc</groupId>
      <artifactId>grpc-proto</artifactId>
      <version>${project.version}</version>
      <exclusions>
        <exclusion>
          <groupId>*</groupId>
          <artifactId>*</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
    <dependency>
      <groupId>io.stargate</groupId>
      <artifactId>sgv2-service-common</artifactId>
      <version>${project.version}</version>
      <exclusions>
        <exclusion>
          <groupId>*</groupId>
          <artifactId>*</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
 ```
 
I get all the common classes, for sure if I use anything that depends on the Dropwizard or something else I will get a runtime error, but for now it's good.. And if we create a dedicated common for the Quarkus or framework-agnostic it would be better..

Reply:
That doesn't solve the problem, unit tests are still failing.